ledger/participant-integration-api/src/main/resources/db/migration/postgres-appendonly/V118__add_transaction_metering.sql

```diff
@@ -0,0 +1,20 @@
+--  Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+--  SPDX-License-Identifier: Apache-2.0
+
+SELECT 'Add Transaction Metering: Creating table...';
+
+CREATE TABLE transaction_metering (
```

Comment:
I'd suggest to add limits to the `VARCHAR`s to communicate the expected upper bound. I'd use `512` for both offsets and application_id, as they are `LedgerString`s, which are currently lenght-limited to 256 chars. So 512 leaves ample space for growth, but still makes it explicit that there's a limit.

Reply:
@meiersi-da I would not mind having that unbounded as in other places, and fixing only as the big overhaul comes with the API limits, but I see also value making a move in the right direction with limit. I have no strong opinion on this matter.
@simonmaxen-da what do you think?

Reply:
Fixed at 512.